src/adapters/TelnetAdapter.ts

```diff
@@ -1291,7 +1301,7 @@ export class RequestPipeline {
             let request = {
                 executeCommand: executeCommand,
                 onComplete: (data: string) => {
-                    util.logDebugFenced(`Command ${commandId} execute ${JSON.stringify(command)} result`, data);
+                    util.logDebugFenced(`Command ${commandId} result`, data);
```

Comment:
The command already has an id, and the previous log statement with this ID showed the full command to execute. So we don't need to clutter the log with that same text again. just match up the `commandId`.